ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -682,6 +682,37 @@ public class StoreConfig {
   public final boolean storeReshuffleDisksOnReorder;
   public final static String storeReshuffleDisksOnReorderName = "store.reshuffle.disks.on.reorder";
 
+  /**
+   * Decide Replication Protocol For Hydration Of Newly Added Replicas
+   */
+  public static final String SERVER_REPLICATION_PROTOCOL_FOR_HYDRATION = "server.replication.protocol.for.hydration";
```

Comment:
Should be renamed to SERVER_REPLICATION_PROTOCOL_FOR_REPLICATION

Reply:
We are only dependent on file copy protocol for initial hydration. Bootstrap can confuse it with the Helix state, even though we are doing this in the OFFLINE state only.